pitest-maven/src/main/java/org/pitest/maven/PitMojo.java

```diff
@@ -47,7 +47,7 @@
    * @parameter expression="${targetClasses}"
    *
    */
-  protected ArrayList<String>         targetClasses;
+  private List<String>         targetClasses;
```

Comment:
I think we did not talk about this specific expectation. I'd assume both.

But maybe we can be clear on #202 to meet our both expactations ;).